README.md

```diff
@@ -10,3 +10,36 @@ CodeFlare Stack Compatibility Matrix
 | CodeFlare-SDK                | v0.4.4  |
 | InstaScale                   | v0.0.4  |
 | KubeRay                      | v0.5.0  |
+
+## Release process
+
+Prerequisite:
+- Build and release [MCAD](https://github.com/project-codeflare/multi-cluster-app-dispatcher)
+- Build and release [InstaScale](https://github.com/project-codeflare/instascale)
+- Build and release [CodeFlare-SDK](https://github.com/project-codeflare/codeflare-sdk)
+
+Release steps:
+1. Invoke [tag-and-build.yml](https://github.com/project-codeflare/codeflare-operator/actions/workflows/tag-and-build.yml) GitHub action, this action will create a repository tag, build and push operator image.
+
+2. Check result of [tag-and-build.yml](https://github.com/project-codeflare/codeflare-operator/actions/workflows/tag-and-build.yml) GitHub action, it should pass.
+
+3. Update CodeFlare Stack Compatibility Matrix in operator README.
+
+4. Update InstaScale and MCAD versions:
+- in [Makefile](https://github.com/project-codeflare/codeflare-operator/blob/main/Makefile#L12-L16).
```

Comment:
The stable link may be better: https://github.com/project-codeflare/codeflare-operator/blob/02e14b535b4f7172b0b809bcae4025008a1a968b/Makefile#L12-L16.